applications/vanilla/controllers/class.categoriescontroller.php

```diff
@@ -109,12 +109,12 @@ public function Archives($Category, $Month, $Page = FALSE) {
    /**
     * "Table" layout for categories. Mimics more traditional forum category layout.
     */
-   public function Table($Category = '') {
+   public function Table($Category = '', $isRoot = TRUE) {
```

Comment:
We can't add parameters to control methods like this. Remember that controller methods concert to url and an ?isRoot=true is not appropriate. Don't you know we are in the root if `!$Category`?